packages/ui/components/NetworkOptions.vue

```diff
@@ -1,31 +1,29 @@
 <template>
-  <div :class="type">
-    <div v-for="option in options" :key="option" class="option">
-      <transition name="fill" mode="out-in">
-        <nuxt-link
-          :key="
-            ((selected[0] && selected[0].chain.toLowerCase()) || 'ethereum') ===
-            option.toLowerCase()
-          "
-          class="nav-link"
-          :to="
-            localeRoute({
-              name: 'network',
-              params: {
-                network: option.toLowerCase(),
-              },
-            })
-          "
+  <div class="nav-container" :class="type">
+    <div class="networks">
+      <div v-for="option in mainOptions" :key="option.name" class="option">
+        <LazyNetworkLink :name="option.name" :svg="option.logo" />
+      </div>
+    </div>
+    <transition name="dropdown" class="dropdown">
+      <div v-if="showAll" class="networks">
+        <div
+          v-for="option in filteredOptions"
+          :key="option.name"
+          class="option"
         >
-          {{ option }}
-        </nuxt-link>
-      </transition>
+          <LazyNetworkLink :name="option.name" :svg="option.logo" />
+        </div>
+      </div>
+    </transition>
+    <div v-if="type === 'sidebar'" class="show-more-btn" @click="toggleShowAll">
+      <p v-if="showAll">↑ {{ $t('less_networks') }} {{ networksLeft }} ↑</p>
+      <p v-if="!showAll">↓ {{ $t('more_networks') }} {{ networksLeft }} ↓</p>
```

Comment:
I used these stylish arrows in the mockup, but I think the pointy triangles is what most people use in similar contexts:
```suggestion
      <p v-if="showAll">▲ {{ $t('less_networks') }} {{ networksLeft }} ▲</p>
      <p v-if="!showAll">▼ {{ $t('more_networks') }} {{ networksLeft }} ▼</p>
```
I'd probably wrap them in a `<span>` so that we can adjust their styling separately to the text part, as I bet they could probably use a slightly lighter color for them not to look too bold.